clients/drcachesim/tracer/instru.cpp

```diff
@@ -185,36 +185,30 @@ instru_t::instr_to_prefetch_type(instr_t *instr)
 bool
```

Comment:
The description says #4382 but I don't see any disasm changes here (if there were: I would vote for a separate PR) -- is that a typo?

Reply:
> The description says #4382 but I don't see any disasm changes here (if there were: I would vote for a separate PR) -- is that a typo?

The reason I added #4382 is because there was discussion of splitting out SYS:
https://github.com/DynamoRIO/dynamorio/issues/4382#issuecomment-664596105
https://github.com/DynamoRIO/dynamorio/issues/4382#issuecomment-668720692

Reply:
OK.